css/MLS.css

```
@@ -33,9 +33,9 @@ a:hover { text-decoration: underline; }
  * font set to monospace (the font probably only matters for font size selection in the case of math content).
  * 13/16 = 0.8125
  */
-/*.ltx_font_typewriter { font-size:123%; }*/
-.ltx_Math { font-size: 81.25%; }
-.ltx_font_typewriter .ltx_Math { font-size: 100%; }
+.ltx_font_typewriter { font-size:110%; }
+.ltx_Math { font-size:110%; }
+/*.ltx_font_typewriter .ltx_Math { font-size: 100%; }*/
```

I find the math just a tad too big, and for me it actually looks better this way:
```suggestion
/*.ltx_Math { font-size:100%; }*/
/*.ltx_font_typewriter .ltx_Math { font-size: 110%; }*/
```

---

While the above looks good for me in Chrome and Safari, the normal math looks too small in Firefox.  I suppose one would need to take MathJax or not into account for best results…

---

Here's a concrete proposal that I think looks OK both in Chrome (MathJax) and Firefox:
```suggestion
.ltx_Math { font-size: 120%; }
.MathJax .ltx_Math { font-size: 100%; }
.ltx_font_typewriter { font-size:110%; }
.ltx_font_typewriter .ltx_Math { font-size: 120%; }
.ltx_font_typewriter .MathJax .ltx_Math { font-size: 110%; }
```

I'm sure it can be fine-tuned further, but it's a starting point allowing us to control MathJax sizes separately.